ui/app/serializers/secret-engine.js

```diff
@@ -80,10 +80,18 @@ export default ApplicationSerializer.extend(EmbeddedRecordsMixin, {
   serialize(snapshot) {
     const type = snapshot.record.engineType;
     const data = this._super(...arguments);
+    const WIF_SECRET_ENGINES = ['aws'];
```

Comment:
It might be good to move this into a util at some point, since I think I've noticed some other places where we're checking for if an engine is WIF